### ISSUE TITLE
404: Not Found

### ISSUE THREAD

**Issue by frenchja** [julianlam/nodebb-plugin-emailer-local#4]
Hi, I installed nodebb-plugin-emailer-local and there were no warnings or errors in the npm output.  However, when I click on the plugin in the Admin panel I get a 404:

```
Not Found
You seem to have stumbled upon a page that does not exist. Return to the home page.
```

I didn't see anything in my socketio.log.  How can I help you troubleshoot this?

**Comment by Rick83600** [comment]
I have the same problem any issues ? :)

**Comment by fdemiramon** [comment]
The plugin registered is too old for the current version of Nodebb. Especially the routes that are configured differently. Here is a fork that might work: https://github.com/psychobunny/nodebb-plugin-emailer-local

**Comment by Couby** [comment]
How do you install this fork ?
In the readme you have the same install process described.

**Comment by Couby** [comment]
I think I found the answet on NodeBB forum :
npm install git://github.com/psychobunny/nodebb-plugin-emailer-local

**Comment by julianlam** [comment]
Fixed in v0.2.0, grab it via plugins page (in about an hour from now).
